recipe/meta.yaml

```diff
@@ -1,42 +1,48 @@
 {% set name = "fast-pt" %}
-{% set version = "3.1.1" %}
+{% set version = "3.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/JoeMcEwen/FAST-PT/archive/v{{ version }}.tar.gz
-  sha256: 6b444fe05817b0f5d2d252f0868bf54f39661f41ca967a6609f1be0e995666ae
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/fast_pt-{{ version }}.tar.gz
+  sha256: ec4cfdb4a352b300ca7ec2f313a73c5676ec61ef665345b0fc37a258f5ecbd5d
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.5
```

Comment:
Please use the python_min syntax in the hints.

Reply:
I added, let me know if I did it right or not.